### ISSUE TITLE
Create and publish R&S IRIS FAQ for messaging about its retirement

### ISSUE THREAD

**Issue by bethpottsVADEPO** [department-of-veterans-affairs/va.gov-team#29176]
## Issue Description
The product team working on the AVA form and the retirement of IRIS wants an FAQ where they can point users from a banner link on the IRIS home page. They provided some text that we need to adjust for our voice, style, and plain language standards. 

## Initial content (to be edited further in drupal once the article is created)

**H1: IRIS and the GI Bill Help portal are no longer available**

We’ve recently retired our Information Routing & Inquiry Service (IRIS) and the GI Bill Help portal. We have a new online service called Ask VA (AVA) that you can use to submit questions to us. AVA has replaced IRIS and the GI Bill Help portal. To use AVA, you'll need to sign in to VA.gov with ID.me.

CTA BUTTON 1: Contact us using our new online service
CTA BUTTON 2: Create an ID.me account

**H2: What is Ask VA (AVA)?** 
AVA is the system for all Veterans and members of the Veteran community to submit questions, compliments, concerns, and recommendations to the Department of Veteran Affairs. AVA allows submitters to directly interact with the different administrations and receive verified, accurate, and timely responses to their inquiries. 

**H2: Why is IRIS changing to Ask VA(AVA)?** 
The IRIS/GI Bill systems are being upgraded to offer a modernized process for you, the submitter, and the VA Administrations reviewing and answering your questions. 
 
**What is changing with the transition from IRIS to AVA?** 
To receive the best experience using AVA, you will want to create an ID.me account. The ID.me account will track all inquiries you submit to VA, store your information for quick inquiry creation and ensure secure messaging between you and the Department of Veteran Affairs 
(ID.me link here)
GI Bill Help will be using the same system as all other administrations 

---
## Tasks
- [x] Create drupal FAQ article, which will have multiple Q&As.
- [x] Create FAQ with initial content 
- [ ] Add more Q&As once content is ready that will address creation of an ID.me account for using AVA.
- [x] Review and publish page

## Acceptance Criteria
- [x] FAQ is ready and queued up for launch date. 
- [x] FAQ is on live site

**Comment by lunascoop** [comment]
@bethpottsVADEPO this article is ready for review.

- (Multiple FAQ Version) IRIS and the GI Bill Help portal are no longer available: http://preview-prod.vfs.va.gov/preview?nodeId=34313
- (About detailed page version) IRIS and the GI Bill Help portal are no longer available: http://preview-prod.vfs.va.gov/preview?nodeId=34309

**Comment by bethpottsVADEPO** [comment]
Update: I'm continuing to make updates to the FAQ as I communicate with AVA stakeholders about the details--especially about identity verification. 

I got sign off from Education Service (Joseph Preisser).

Next: I need to send the latest version to an ID.me SME for approval.

**Comment by bethpottsVADEPO** [comment]
Got a SME review from Kyle Fairbanks with ID.me. Approved.

Communicated with Chris Clemens on the AVA team and we agreed to publish today.

**Comment by RLHecht** [comment]
This page is published. Moving to validate.

**Comment by RLHecht** [comment]
The new article is live. Closing ticket. We will open a new ticket if more questions get added to the FAQ.
